packages/bot-web-ui/src/components/quick-strategy/quick-strategy.tsx

```diff
@@ -101,6 +109,12 @@ const FormikWrapper: React.FC<TFormikWrapper> = observer(({ children }) => {
                             }
                             min_error = getErrorMessage('MIN', min);
                         }
+                        if (should_validate && field.name === 'last_digit_prediction') {
+                            max = 0;
+                            max = 9;
+                            max_error = getErrorMessage('MAX', max, 'LAST_DIGIT_PREDICTION');
```

Comment:
@maryia-matskevich-deriv it should be min = 1 and max = 9

Reply:
@farabi-deriv no, minimum value is `0`, so we have to use it
<img width="1723" alt="Screenshot 2023-12-21 at 09 34 10" src="https://github.com/binary-com/deriv-app/assets/103181650/8cb2af32-7e66-4b6f-893f-827ec8490dc5">

Reply:
@maryia-matskevich-deriv Ok so min should be 0. But you set both as max value. max = 0 and then max = 9. Should be min.

Reply:
oh yes, thank you, will fix it

Reply:
fixed✅